### ISSUE TITLE
ELI5 installation instructions

### ISSUE THREAD

**Issue by Joseph-R** [sequenceiq/ambari-shell#43]
Hey guys,

So we're using CloudBreak in a PoC and probably in staging and production soon.  We like Ambari-shell, but have given up on [docker-ambari](https://github.com/sequenceiq/docker-ambari) for our local development environment.  So we would like to install Ambari Shell locally on your typical HDP-on-VM scenario.

I'm trying to understand how to get Ambari Shell installed based on [your instructions](https://github.com/sequenceiq/ambari-shell#installation-and-usage) and I'm just not getting it.
1.  So I need to download a JAR?  Or compile a JAR?
2.  Where do I get said JAR?
3.  How can I go about automating this?

If you could sort of walk me through it here or point me to some better docs, that would be great.  I'm happy to submit a PR or something to update the README.md.

Regards,
Joe Reid

**Comment by lalyos** [comment]
All jars are delivered to our maven repos:

- maven.sequenceiq.com/snapshots for snapshots
- maven.sequenceiq.com/releases for releases

## references:

Latest released jar: http://maven.sequenceiq.com/release/com/sequenceiq/ambari/ambari-shell/1.5.1/ambari-shell-1.5.1.jar

To get the latest snapshot: use the script https://github.com/sequenceiq/ambari-shell/blob/master/latest-snap.sh

### Metadata
- http://maven.sequenceiq.com/releases/com/sequenceiq/ambari/ambari-shell/maven-metadata.xml
- http://maven.sequenceiq.com/snapshots/com/sequenceiq/ambari/ambari-shell/maven-metadata.xml 

## building:

```
./gradlew build
```
